src/index.js

```diff
@@ -69,7 +69,7 @@ const dirTree = folderName => {
 
   if (fs.existsSync(`${folderName}/${configFileName}`)) {
     try {
-      const config = yaml.safeLoad(
+      const config = yaml.load(
```

Comment:
Fixes `Failed to parse .indexer.yaml Error: Function yaml.safeLoad is removed in js-yaml 4. Use yaml.load instead, which is now safe by default.`